src/api.ts

```diff
@@ -26,7 +26,7 @@ export interface ClientConfig {
 
 export function createClient(config: ClientConfig): ApiClient {
 	return new ApiClient({
-		apiPrefix: `${config.apiEndpoint}/v4/`,
+		apiPrefix: `${config.apiEndpoint}/v5/`,
```

Comment:
Nit: we should fix the comment that mentions v4 above too

Reply:
True, fixed.